tests/unit/resource/resource_tests.cpp

```diff
@@ -37,34 +37,3 @@ TEST(DefaultMemoryResource, AllocateDeallocate)
 
   alloc->deallocate(pointer);
 }
-
-TEST(DefaultMemoryResource, GetSize)
```

Comment:
This doesn't work now that MemoryResources don't track their own introspection data.